trezorlib/client.py

```diff
@@ -834,6 +834,80 @@ def mosaic_supply_change_to_proto(mosaic_supply_change):
 
         return self.call(msg)
 
+    #
+    # Lisk functions
+    #
+
+    @field('address')
+    @expect(proto.LiskAddress)
+    def lisk_get_address(self, n, show_display=False):
+        n = self._convert_prime(n)
+        return self.call(proto.LiskGetAddress(address_n=n, show_display=show_display))
+
+    @expect(proto.LiskPublicKey)
+    def lisk_get_public_key(self, n, show_display=False):
+        n = self._convert_prime(n)
+        return self.call(proto.LiskGetPublicKey(address_n=n, show_display=show_display))
+
+    @expect(proto.LiskMessageSignature)
+    def lisk_sign_message(self, n, message):
+        n = self._convert_prime(n)
+        message = normalize_nfc(message)
+        return self.call(proto.LiskSignMessage(address_n=n, message=message))
+
+    def lisk_verify_message(self, pubkey, signature, message):
+        message = normalize_nfc(message)
+        try:
+            resp = self.call(proto.LiskVerifyMessage(signature=signature, public_key=pubkey, message=message))
+        except CallException as e:
+            resp = e
+        if isinstance(resp, proto.Success):
```

Comment:
please change this to `return isinstance(...)`
(yes, all other verify_message functions have this. that will be changed, let's not put it into new code)

Reply:
@matejcik done